internal/pkg/manifest/workload.go

```diff
@@ -115,6 +117,61 @@ func (i *Image) cacheFrom() []string {
 	return i.Build.BuildArgs.CacheFrom
 }
 
+// ImageOverride holds fields that override Dockerfile image defaults.
+type ImageOverride struct {
+	EntryPoint EntryPointOverride `yaml:"entrypoint"`
+	Command    CommandOverride    `yaml:"command"`
+}
+
+type EntryPointOverride StringSliceOrString
```

Comment:
nice! this is so smart 😁 

1. Can we make `StringSliceOrString` private? I don't think that type will be used in the `stack` pkg.
```suggestion
type EntryPointOverride stringSliceOrString
```
2. Can we also add [doc comments](https://github.com/golang/go/wiki/CodeReviewComments#doc-comments) to these two types since they're exported.